pkg/java/java.go

```diff
@@ -49,6 +49,22 @@ func (o *Opts) Executable() string {
 	return pathx.Normalize(o.HomeDir) + "/bin/java"
 }
 
+func (o *Opts) Env() []string {
+	javaDir := pathx.Abs(o.HomeDir)
+	javaPath := javaDir + "/bin"
+	others := osx.EnvVarsWithout("PATH", "JAVA_HOME")
+	return append([]string{
+		"PATH=" + javaPath + ":" + os.Getenv("PATH"),
```

Comment:
clue of change is to prepend configured Java path by AEMC/aem.yml before using configured by default by OS/jenv etc

Reply:
this is to enforce AEM control scripts crx-quickstart/start|stop|status to be using configured Java version; AEMC tries to isolate Java installed globally from the one/dedicated to AEM / Oracle

Reply:
this technique eliminates many real-life errors like "works on my machine" ... "everything looks same but does not work" .... yeah but different java ;P